client/js/content.js

```diff
@@ -43,19 +43,31 @@ const getRatingDivElement = (title, rating) => {
 const getInnerHTML = (imdbRating, rottenTomatoRating) => {
     const innerHTML = `
     <div>
-    <p style="background-color: red;
-        z-index: 2;
-        margin-top: 0px;
-        position: absolute;
-        top:0; right:0;"
-    >IMDB: ${imdbRating}</p>
-    <p style="background-color: red;
-    z-index: auto;
-    margin-top: 0px;
-    position: absolute;
-    top:0; left:0;"
->Rotten Tomato: ${rottenTomatoRating}</p>
-    </div>
+    <p>
+    <span style = "box-sizing: border-box;
```

Comment:
ideally this shall go to a css file and we shall put class names here. I can do it in next PR or if you are handling it then its great as well